src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -29,7 +29,8 @@ import scala.util.{Random, Try}
 class NetworkController(ergoSettings: ErgoSettings,
                         peerManagerRef: ActorRef,
                         scorexContext: ScorexContext,
-                        tcpManager: ActorRef
+                        tcpManager: ActorRef,
+                        messageHandlersPartial: ActorRef => Map[MessageCode, ActorRef]
```

Comment:
There's no need in such flexibility probably, handlers can be just hardcoded in NetworkController

Reply:
The problem is that code becomes very mess, if I try to create these actors inside NetworkController.
So in my opinion partial construction works better for readability.